common/src/profile_update_channel.rs

```diff
@@ -0,0 +1,62 @@
+use std::sync::Arc;
+
+use futures::channel::oneshot;
+use once_cell::sync::Lazy;
+use tokio::sync::Mutex;
+use warp::crypto::DID;
+
+use crate::{
+    state::Identity,
+    warp_runner::{MultiPassCmd, WarpCmd},
+    WARP_CMD_CH,
+};
+
+pub struct ProfileUpdateChannel {
+    pub tx: tokio::sync::mpsc::UnboundedSender<ProfileUpdateAction>,
+    pub rx: Arc<Mutex<tokio::sync::mpsc::UnboundedReceiver<ProfileUpdateAction>>>,
+}
+
+pub static PROFILE_CHANNEL_LISTENER: Lazy<ProfileUpdateChannel> = Lazy::new(|| {
+    let (tx, rx) = tokio::sync::mpsc::unbounded_channel();
+    ProfileUpdateChannel {
+        tx,
+        rx: Arc::new(Mutex::new(rx)),
+    }
+});
```

Comment:
Although previously noted, but this should be localized in some manner than being stored in a lazy imo. Doesnt have to happen in this PR though.